src/@ionic-native/plugins/appodeal/index.ts

```diff
@@ -22,9 +22,9 @@ import { Injectable } from '@angular/core';
  */
 @Plugin({
   pluginName: 'Appodeal',
-  plugin: 'https://github.com/appodeal/appodeal-cordova-plugin',
+  plugin: 'https://github.com/appodeal/appodeal-cordova-plugin.git',
```

Comment:
I want to unify the plugins if they download the files from GitHub.
If you try to download the files without .git, GitHub redirects to the .git file.